Sniffs/UpdateURISniff.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace ET\ElegantThemes\Sniffs;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class UpdateURISniff implements Sniff {
+
+	/**
+	 * Returns the token types that this sniff is interested in.
+	 * @return array(int)
```

Comment:
@lanresmith Should it be an `array(string)`?

Reply:
The PHP token constants are `int`s; you can do a `var_dump` of e.g. `T_COMMENT` to see.

Reply:
I was talking about `T_DOC_COMMENT_STRING`. Judging from the constant name, it should be string. But, I just remember something and since you said  `T_COMMENT` is integer, I think the correct one should be `int[]|string[]` which means collection of integer or string. Ref: https://docs.phpdoc.org/guide/guides/types.html search for `string[]`.

Reply:
@lanresmith Aha.. `mixed` is more simple. Let's use `mixed[]`!

**NOTES:**
Even though `(int|string)[]` is `OR`, it's still correct when it contains both of `int` and `string` because in math logic `OR` is `true` when ONE of them or even BOTH of them is `true`. For example, `3|'lala'` of `(int|string)[]` is `true` in this case. And also, this is what the documentation says about [array multiple types](https://docs.phpdoc.org/3.0/guide/references/phpdoc/types.html#arrays):

> specified containing multiple types, the Type definition informs the reader of the type of each array element. Each element can be of any of the given types. Example: @return (int|string)[]

But, let's use `mixed[]` because it's more simple.